chromadb/utils/embedding_functions.py

```diff
@@ -442,6 +483,21 @@ def _forward(self, documents: List[str], batch_size: int = 32) -> npt.NDArray:
             all_embeddings.append(embeddings)
         return np.concatenate(all_embeddings)
 
+    def _get_token_counter(self) -> Any:
+        """
+        This function creates a tokenizer with doubled max length to avoid truncation to max_length which bypasses
```

Comment:
I don't understand what this means. How does the `check_large_inputs` function get bypassed?

Reply:
In constructing the tokenizer (not the one in this function), we set truncation mode and max tokens, which automatically truncates any input (no warning) that goes beyond the token limit. This function intends to create a tokenizer with a higher token limit so that we can detect when the user's input will get truncated by the model's tokenizer.

Reply:
how does a tokenizer with a higher limit allow us to do that? can't we just check if max tokens is reached on the existing tokenizer?

Reply:
We can, but what if someone chunks text with 256 tokens exactly (perhaps using the same tokenizer)? The reason we have a secondary tokenizer is to check just this border condition where requested tokens go over the maximum limit, which won't be detectable with the existing tokenizer as output gets truncated.